src/openmrs-fetch.ts

```diff
@@ -0,0 +1,145 @@
+import isPlainObject from "lodash-es/isPlainObject";
+
+export function openmrsFetch(url: string, fetchInit: FetchConfig = {}) {
```

Comment:
How should we document? Function comments or something else?

Reply:
Typescript definition will be the thing that helps many developers. But also I think all our documentation will be in the OpenMRS wiki, too

Reply:
The basic thing to note is that openmrsFetch behaves like the browser’s built in fetch, except with a couple extra nice things such as automatically stringifying request body and calling response.json() for you

Reply:
Yeah, I feel like we should document that behavior (and the null Accept thing, etc). It’s not evinced by the TS definition. Can we commit to maintaining function documentation in a wiki? I feel like it would be easier to maintain documentation as function comments and e.g. generate markup to copy-paste into the wiki from there.

Reply:
What is the way forward here?

Reply:
I don't think function comments should be used as real documentation for OpenMRS. js-doc exists, but I think just using the Atlassian wiki is the preferred method.